lib/WebpackPresetsApply.js

```diff
@@ -0,0 +1,20 @@
+const path = require("path");
+const PresetsApply = require("./PresetsApply");
+
+const pkgPath = path.join(process.cwd(), "package.json");
+const pkg = require(pkgPath);
+const presets = Object.keys(pkg.devDependencies)
+	.filter(pkgName => pkgName.startsWith("webpack-preset-"))
+	.map(pkgName => path.resolve(process.cwd(), "node_modules", pkgName))
```

Comment:
This approach will not work when the presets are hoisted up somewhere higher in the directory structure, are installed globally or when Yarn PnP is used. Maybe enhanced-resolve could be used instead?